tests/test_oidcop.py

```diff
@@ -214,27 +221,19 @@
             ],
             "userinfo_encryption_alg_values_supported": [
               "RSA-OAEP",
-              "RSA-OAEP-256",
-              "A192KW",
-              "A256KW",
-              "ECDH-ES",
-              "ECDH-ES+A128KW",
-              "ECDH-ES+A192KW",
```

Comment:
why do you want to remove elliptical encryption support? Is a good encryption method for me :)